packages/farms/src/index.ts

```diff
@@ -46,8 +46,6 @@ export function createFarmFetcher(multicallv2: MultiCallV2) {
       farmsWithPrice,
       poolLength: poolLength.toNumber(),
       regularCakePerBlock: +regularCakePerBlock,
-      totalRegularAllocPoint,
-      cakePerBlock,
```

Comment:
removed this as it seemed to only be needed in woodys custom useFarmsV2Data hook whch used fetcher.FetchFarms to get these two.since all other instances of fetch.fetchFarms dont need these two i removed (this was the old state of this code anyway before this PR)